docs/markdown/Getting Started/getting-started/installation.md

```diff
@@ -6,40 +6,50 @@ createdAt: "2020-02-21T17:44:53.022Z"
 updatedAt: "2022-07-12T00:00:00.000Z"
 ---
 
-Most installations can use our one-step setup script to get a minimal version of Pants set up. If you need to make use of an advanced or non-standard setup, see our [manual installation](doc:manual-installation) page.
-
-Using our one-step setup script
--------------------------------
-
-Pants has a launch script (called `./pants`) that handles downloading, bootstrapping, and upgrading Pants, which you need to save at the root of your repository. 
-
-Pants also needs a `pants.toml` file, where you will eventually add all of the configuration needed to run testing, linting, and formatting rules. For now, it just needs to specify the version of Pants that you want to use. 
-
-To streamline this, we provide a script that will create a minimal `pants.toml` file specifying the latest released version of Pants, download, and run the `./pants` launch script.
+To install the `pants` binary use:
 
 ```
-/bin/bash -c "$(curl -fsSL https://static.pantsbuild.org/setup/one_step_setup.sh)" 
+/bin/bash -c "$(curl -fsSL https://static.pantsbuild.org/setup/pantsup.sh)" 
```

Comment:
This assumes `~/bin` being on `PATH` is popular enough or else it assumes folks read well when the script warns them it is not. Are you comfortable with that?

Reply:
That's a good point. I'll make sure this is documented.